### PR TITLE
create RC docs for Fx59 + Fennec59

### PR DESCRIPTION
**Pull request by escapewindow**

### CODE REVIEW

File: docs/release-promotion/mobile/howto-rc.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+INFO: writing to wiki file: /Users/sfraser/github/mozilla-releng/releasewarrior-data/inflight/fennec/fennec-release-rc-60.0.md
+INFO: writing to corsica file: /Users/sfraser/github/mozilla-releng/releasewarrior-data/index.html
+INFO: committing changes with message: fennec 60.0 - updated inflight tasks. Resolved ('3',)
+```
+
+Remember to run `git push` in the `releasewarrior-data` repository, if needed.
+
+
+***
+Last checked: 2017-12-19 by sfraser
```
Comment (MihaiTabara) [inline docs/release-promotion/mobile/howto-rc.md]: Remove this line?

File: releasewarrior/templates/firefox/release-rc.json.tmpl [hunk 2]
```diff
@@ -17,9 +17,9 @@
                 "resolved": false
             },
             {
-                "alias": "beta",
+                "alias": "ship-rc",
                 "description": "publish in Balrog on beta channel",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Release-Promotion-Tasks#publish-the-release",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto.md#ship-rc",
```
Comment (MihaiTabara) [inline releasewarrior/templates/firefox/release-rc.json.tmpl]: This url seems wrong. Also, seems to point to existing #howto/ship-rc when you added separate RC docs in this PR https://github.com/mozilla-releng/releasewarrior-2.0/pull/68/files#diff-77ab09a8bc720f91b95634e2562879b1R89

File: releasewarrior/templates/firefox/release-rc.json.tmpl [hunk 3]
```diff
@@ -37,13 +37,13 @@
             {
                 "alias": "mirrors",
                 "description": "pushed to mirrors/releases",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Release-Promotion-Tasks#push-artifacts-to-releases-directory",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto.md#push",
```
Comment (MihaiTabara) [inline releasewarrior/templates/firefox/release-rc.json.tmpl]: URL seems wrong, http:// in the middle of the url. Wrong interpolation/search-replace maybe?

File: releasewarrior/templates/firefox/release-rc.json.tmpl [hunk 4]
```diff
@@ -37,13 +37,13 @@
...
                 "resolved": false
             },
             {
-                "alias": "publish",
+                "alias": "ship",
                 "description": "publish release tasks",
-                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/wiki/Release-Promotion-Tasks#publish-the-release",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto.md#ship",
```
Comment (MihaiTabara) [inline releasewarrior/templates/firefox/release-rc.json.tmpl]: Same as above

**Comment by MihaiTabara** [comment]
Also, thank you for this! Nice to see this coming along for Fennec as well! 👍

**Comment by escapewindow** [comment]
Fixed, thanks!
I think a lot of the templates are still pointing at the rw2.0 wiki. =\

**Comment by MihaiTabara** [comment]
Some of the wrong hrefs were addressed by @lundjordan's https://github.com/mozilla-releng/releasewarrior-2.0/pull/69 . I'd lean towards fixing on the fly since the wiki is now dead and we have enough docs migrated to make the switch. Feel free to merge whenever you think you're ready with this.

**Comment by escapewindow** [comment]
Hoping we don't have another dot release before EOW... merging.

**Pull request merged**